src/controllers/createUserController.js

```diff
@@ -29,6 +29,7 @@ class CreateUserController extends BaseController {
     }
 
     // TODO: Store the id of that user inside the users cookie (in the controller layer)
+    req.session.userId = user.id;
```

Comment:
yeah, let's make sure to add an assertion that establishes what we're doing here.